certbot/auth_handler.py

```diff
@@ -96,7 +97,7 @@ def handle_authorizations(self, orderr, best_effort=False):
 
     def _choose_challenges(self, aauthzrs):
         """Retrieve necessary challenges to satisfy server."""
-        logger.info("Performing the following challenges:")
+        logger.info("Following challenges are required:")
```

Comment:
nit: I personally like the wording as is and I think it's still accurate as we're not performing the challenges that have already been validated.